src/hooks/sdk/useSelectAudioInputDevice.tsx

```diff
@@ -6,11 +6,13 @@ import { useCallback } from 'react';
 
 import { useMeetingManager } from '../../providers/MeetingProvider';
 
-export const useSelectAudioInputDevice = () => {
+export const useSelectAudioInputDevice = (): ((
+  device: Device | AudioTransformDevice | null
```

Comment:
isn't this the same as `AudioInputDevice` from JS SDK? Why dont we just use it?

Reply:
We should. I missed this.

Reply:
I missed it in the second commit but updated it in the third commit. No change is required.